features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,20 @@
-# put step definitions in this file
\ No newline at end of file
+When(/^I visit the index site$/) do
+   visit '/'
+end
+
+Then(/^I should see "([^"]*)"$/) do |arg1|
+   expect(page).to have_content arg1
+end
```

Comment:
You should rename your variables to something logical - here it should be `content` or `text`.